### DISCUSSION TITLE
Graphic card choose

### DISCUSSION THREAD

**Discussion by 666damn** [rustdesk/rustdesk#10066] [Feature Request]
### Bug Description

My laptop has Intel HD Graphics which is with CPU and rtx3060, but everytime I connect it, it automatically pick Intel HD graphics, is there any way to use my 3060 or I can only ban my Intel Graphics in BIOS

### How to Reproduce

Add a bar for pick graphic card

### Expected Behavior

Have a bar to pick graphic card

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 10

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.2

### Screenshots

![捕获](https://github.com/user-attachments/assets/39886c6b-e9ee-4d17-a3c3-cc8159d836eb)

**Comment by rustdesk** [comment]
@21pages I mark this as feature request, please put in your priority, but low priority.

**Comment by xlionjuan** [comment]
https://www.asus.com.cn/support/faq/1044213/
